.github/workflows/nightly-build.yml

```diff
@@ -0,0 +1,107 @@
+name: Tekton Dashboard Nightly Build
+on:
+  schedule:
+    # Run at 03:00 UTC daily
+    - cron: "0 3 * * *"
+  workflow_dispatch:
+    inputs:
+      kubernetes_version:
+        description: 'Kubernetes version to test with'
+        required: false
+        default: 'v1.33.x'
```

Comment:
Can we omit the default value here so it's just defined in 1 place? i.e the env var below
Or do we have to specify a default value for an optional input?